aws_lambda_powertools/utilities/parameters/dynamodb.py

```diff
@@ -0,0 +1,164 @@
+"""
+Amazon DynamoDB parameter retrieval and caching utility
+"""
+
+
+from typing import Dict, Optional
+
+import boto3
+from boto3.dynamodb.conditions import Key
+from botocore.config import Config
+
+from .base import BaseProvider
+
+
+class DynamoDBProvider(BaseProvider):
+    """
+    Amazon DynamoDB Parameter Provider
+
+    Parameters
+    ----------
+    table_name: str
+        Name of the DynamoDB table that stores parameters
+    key_attr: str, optional
+        Hash key for the DynamoDB table
+    value_attr: str, optional
+        Attribute that contains the values in the DynamoDB table
+    config: botocore.config.Config, optional
+        Botocore configuration to pass during client initialization
+
+    Example
```

Comment:
**nitpick:** Could you add one more line in the first example showing how to access the returned parameter?

That way it's clearer and you only do that twice - One for `get` and one for `get_multiple`

```
print(f"Parameter value -> { }")
```